client/src/components/Allocations/AllocationsModals/AllocationsManageTeamModal/AllocationsManageTeamModal.module.scss

```diff
@@ -1,7 +1,18 @@
 .root {
+  max-width: 700px;
   .modal-body {
     overflow-y: initial !important;
   }
+  .update-button {
+    display: flex;
+    width: fit-content;
+    border-radius: 0;
+    font: bold 12px Roboto, sans-serif;
+    padding: 0.5rem 1rem;
+    border: 0 !important; /* override `.workbench-content .btn-secondary` */
+    background-color: #9d85ef !important; /* override `.workbench-content .btn-secondary` */
+    color: white !important; /* override `.workbench-content .btn-secondary` */
+  }
```

Comment:
We have common button themes defined elsewhere I believe. Can we use those? Maybe `btn btn-primary` classes?